gitops/environments/base/serviceaccount.yaml

```diff
@@ -1,7 +1,8 @@
+---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: <WL_SERVICE_NAME>
+  name: default-service
```

Comment:
I can't agree that we need to hardcode the service account name. we can have a case when each WL service inside the workload must have its own unique service account.